sdk/samples/storage/blobs/src/blobs_client_example.c

```diff
@@ -37,6 +37,9 @@ static az_span content_to_upload = AZ_SPAN_LITERAL_FROM_STR("Some test content")
 #endif
 
 // Enable logging
+az_log_classification const log_classifications[]
```

Comment:
Good catch. Add comments so someone copying code knows what they  should and shouldn't do.

Reply:
I am hesitant to draw so much of the reader's attention in that instance. We follow a good practice, and I think it is sufficient. The problem is only when we run in a multi-threaded app, but the sample does not have multiple threads. I.e. you came to see the storage sample, and right away you get a couple lines of detailed information on the multi-threaded aspect of logging.